shared/js/background/redirect.es6.js

```diff
@@ -6,6 +6,8 @@ const tabManager = require('./tab-manager.es6')
 const ATB = require('./atb.es6')
 const browserWrapper = require('./$BROWSER-wrapper.es6')
 const settings = require('./settings.es6')
+const webResourceURL = browserWrapper.getExtensionURL('/') + 'web_accessible_resources'
```

Comment:
```suggestion
const webResourceURL = browserWrapper.getExtensionURL('/web_accessible_resources')
```
The underlying call to `chrome.extension.getURL` does URL path construction for us.